include/bl_stash.rst

```diff
@@ -0,0 +1,31 @@
+.. _doc_bl_stash:
+
+Bootloader provisioning data (bl_stash)
```

Comment:
Update this to stash?

Reply:
Applies to this whole file.

Reply:
This was intentional. The thought was: "bl_stash" is the name of the library, "bootloader provisioning data" is what bl_stash operates on.

Do you want to not use the word "provisioning" at all in bl_stash? Can you give an example or two where you change a sentence in the docs?

Reply:
I agree with you that we can use the verb provision alongside a different name for the purse :)